config/initializers/cluster.rb

```diff
@@ -1,6 +1,7 @@
 cluster_yml = YAML.load(File.read('config/cluster.yml'))
 MAINNET_CLUSTER_VERSION = cluster_yml['software_patch_mainnet']
 TESTNET_CLUSTER_VERSION = cluster_yml['software_patch_testnet']
+PYTHNET_CLUSTER_VERSION = cluster_yml['software_patch_pythnet']
```

Comment:
We can also create CLUSTER_VERSION hash which will store clusters for each network